### PR TITLE
Add "testing" section to new environment doc

### PR DESCRIPTION
**Pull request by esoergel**
The new environment QA doc wasn't yet linked anywhere

##### ENVIRONMENTS AFFECTED

None

### CODE REVIEW

**Review by kaapstorm — APPROVED** [review]

File: docs/setup/new_environment.md [hunk 1]
```diff
@@ -329,7 +329,7 @@ If you are using virtualbox, see the [Configuring VirtualBox for testing CommCar
 
 ### Testing your new CommCare Environment
 
-Run the following command to test each of the backing services as described [here](https://dimagi.github.io/commcare-cloud/commcare-cloud/deploy.html#step-3-checking-services-once-deploy-is-complete).
+Run the following command to test each of the backing services as described [here](../commcare-cloud/deploy.md#step-3-checking-services-once-deploy-is-complete).
```
Comment (kaapstorm) [inline docs/setup/new_environment.md:332]: Is github.io going to get this right? Or will fixing this link for github.com break it for github.io?

Comment (esoergel) [reply docs/setup/new_environment.md:332]: I'm pretty sure they both handle this well.  We've got a bunch of relative links like this.  I wasn't sure about the in-page anchor, which I copied from github.io, but I did confirm that it works also on github.com

Comment (esoergel) [reply docs/setup/new_environment.md:332]: Confirming that this works as expected:
https://dimagi.github.io/commcare-cloud/setup/new_environment.html#testing-your-new-commcare-environment

File: docs/setup/new_environment.md [hunk 2]
```diff
@@ -327,6 +327,15 @@ If everything went well, you should now be able to access CommCareHQ from a brow
 
 If you are using virtualbox, see the [Configuring VirtualBox for testing CommCareHQ](../howto/configure-virtualbox.md) page to find the URL which depends on your networking setup.
 
+### Testing your new CommCare Environment
+
+Run the following command to test each of the backing services as described [here](../commcare-cloud/deploy.md#step-3-checking-services-once-deploy-is-complete).
+
+```
+cchq monolith django-manage check_services
```
Comment (proteusvacuum) [inline docs/setup/new_environment.md:335]: nit: every other command on this page is prepended by `$`.

Comment (esoergel) [reply docs/setup/new_environment.md:335]: Thanks!
https://github.com/dimagi/commcare-cloud/pull/4671

**Pull request merged**